velox/exec/tests/AggregationTest.cpp

```diff
@@ -888,7 +889,7 @@ TEST_F(AggregationTest, spill) {
   EXPECT_LT(20 << 20, stats[0].operatorStats[1].spilledBytes);
 }
 
-TEST_F(AggregationTest, spillWithEmptyPartition) {
+DEBUG_ONLY_TEST_F(AggregationTest, spillWithEmptyPartition) {
```

Comment:
How do you decide whether to use TEST_F or DEBUG_ONLY_TEST_F?

Reply:
If we use testvalue, then we need to use DEBUG_ONLY_TEST_F as testvalue is opt out on release build. But we run test in release build in some circle ci setups.

Reply:
Got it. Thank you for the explanation.